R/hipercow.R

```diff
@@ -6,6 +6,7 @@
 ##' @examples
 ##' hipercow()
 hipercow <- function() {
+  hipercow_speak(system.file("comms/moo1.wav", package = "hipercow"))
```

Comment:
Or have hipercow_speak("moo1") and have that call to system file to within that?